crates/rolldown/tests/fixtures/format/iife/external_modules_with_globals/artifacts.snap

```diff
@@ -0,0 +1,20 @@
+---
+source: crates/rolldown_testing/src/case/case.rs
+expression: content
+input_file: crates/rolldown/tests/fixtures/format/iife/external_modules_with_globals
+---
+# Assets
+
+## main.mjs
+
+```js
+(function(node_path) {
+
+const { default: nodePath } = node:path;
```

Comment:
Here has a invalid idenifer 'node:path'

Reply:
I don't know, but Rollup does so. External modules needs the arguments that call the function, which may defined in `output.globals`.

Example code after bundling with Rollup:

```js
(function (node_path) {
	'use strict';



	return node_path.join;

})(node_path);
```

Reply:
The source code is to export the `join` from `node:path`. When I specify the `globals: { 'node:path': 'path' }`, it will generate code in following:

```js
(function (node_path) {
	'use strict';



	return node_path.join;

})(path);
```

Reply:
Sorry for the misunderstanding. I will fix it soon.

Reply:
Fixed.